### DISCUSSION TITLE
How to generate type definitions using 'Type Aliases' instead of 'Interface'

### DISCUSSION THREAD

**Discussion by Armadillidiid** [acacode/swagger-typescript-api#839] [General]
What option could I set in the config to get this output:
```typescript
export type Test = {
  name: string;
  age: number
}
```
Instead of this
```typescript
export interface Test {
  name: string;
  age: number
}
```

**Comment by S0PEX** [comment]
Hey,

did you find any solution for this by any chance?

Regards

Artur

**Comment by Armadillidiid** [comment]
> Hey,
> 
> did you find any solution for this by any chance?
> 
> Regards
> 
> Artur

Sadly, no

**Comment by elioenays** [comment]
I need this too

**Comment by Akronae** [comment]
Need this too, TS interfaces are notoriously a bad choice
